docker/app/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM perl:5.36
+
+WORKDIR "/src"
+
+RUN cpanm URI
+RUN cpanm HTTP::Tiny
+RUN cpanm JSON
+RUN cpanm HTML::TagParser
+RUN cpanm URI::Fetch
+RUN cpanm HTML::TreeBuilder::XPath
+
+CMD perl main.pl
```

https://github.com/otama-jaccy/yapc2023_LT/pull/1/files#diff-8dddb1b04697d1ec855c20a64e11c968db5da4239946d0af93268bba96c601f1R9

で、`use lib '.';` を使っていますが、コードで指定せずに、ここで、
```suggestion
CMD perl -Ilib main.pl
```
とするのがよく見るかなあと思います。